client/scss/components/_standfirst.scss

```diff
@@ -0,0 +1,15 @@
+//* @define standfirst; weak
+
+.standfirst {
+  @include font('HNL2');
+  border-left: 5px solid color('black');
+  padding-left: 1em;
+
+  p {
```

Comment:
Maybe this should be locked down to a single paragraph – `<p class="standfirst">` – rather than allowing for multiple?